### ISSUE TITLE
Organization's extension assets are always being reloaded 

### ISSUE THREAD

**Issue by flbulgarelli** [mumuki/mumuki-laboratory#884]
Regardless they change or not, the organization's js and css is always being reloaded during navigation. 

This is a nasty bug, since it makes the page load times higher. 

This may be related to the fact that turbolinks 5 changed the way of declaring tracked assets: 

### Old

```html
 <script type="text/javascript" .... data-turbolinks-track="true"></script>
 ```

### New

```html
 <script type="text/javascript" .... data-turbolinks-track="reload"></script>
 ```

**Comment by flbulgarelli** [comment]
May be related to #908 and #909

**Comment by julian-berbel** [comment]
This was caused by a problem in the organization's js extension script.

The script was changing elements in the header which would cause turbolinks to trigger a full reload of the page. 

The elements changed where the organization favicon and open graph image. These elements became parametrizable for each organization since PR #892.

We deleted the lines changing those elements in the organization js and set those fields in the organization and the problem ceased. 

To fully fix the problem we'd have to do that for every organization with an extension js script that changes things in the header.
